package.json

```diff
@@ -10,9 +10,11 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@types/mapbox-gl": "^1.8.3",
     "axios": "^0.19.2",
     "bulma": "^0.8.0",
     "formik": "^2.1.4",
+    "mapbox-gl": "^1.9.1",
```

Comment:
I was thinking using like https://github.com/alex3165/react-mapbox-gl#readme, if you can work with this then its fine,